core/include/traccc/ambiguity_resolution/resolution_config.hpp

```diff
@@ -0,0 +1,29 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2025 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// System include(s).
+#include <cstdint>
+#include <limits>
+
+namespace traccc {
+
+/// Configuration struct for ambiguity resolution
+struct resolution_config {
```

Comment:
Nit: Should this be called `ambiguiy_resolution_config` to avoid confusion with configuration for e.g. resolution plots?

Reply:
Makes sense. I changed to `ambiguity_resolution_config`